entryDetail.go

```diff
@@ -612,3 +612,25 @@ func sortEntriesByTraceNumber(entries []*EntryDetail) []*EntryDetail {
 	})
 	return entries
 }
+
+type traceNumbers []string
+
+func (nums traceNumbers) contains(num string) bool {
```

Comment:
For a performance boost, could we use a map instead of a slice?
```go
type traceNumbers map[string]struct{}

func (nums traceNumbers) contains(num string) bool {
  _, ok := nums[num]
  return ok
 }
 ```

Reply:
Yea. I can look at optimizing that. I think I was planning to traverse them and used a `[]string` initially.